cli/onionshare_cli/web/share_mode.py

```diff
@@ -61,10 +61,7 @@ def index(path):
                 and self.download_in_progress
             )
             if deny_download:
-                r = make_response(
-                    render_template("denied.html"),
-                    static_url_path=self.web.static_url_path,
```

Comment:
I think the error thrown was maybe because here `static_url_path` was sent as argument in `make_response` instead of `render_template`. But if removing is the way t we want to go, I am fine with that too, given that it's just a denied page.

Reply:
Oooh, yeah that makes sense. Well, removing it is totally fine anyway I confirmed. There are no static resources on the denied page.